zeebe/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/builder/AbstractUserTaskBuilder.java

```diff
@@ -187,9 +187,12 @@ public B zeebeFormBindingType(final ZeebeBindingType bindingType) {
   }
 
   @Override
-  public B zeebeTaskPriority(final String priority) {
+  public B zeebeTaskPriority(String priority) {
     final ZeebePriorityDefinition priorityDefinition =
         myself.getCreateSingleExtensionElement(ZeebePriorityDefinition.class);
+    if (priority != null) {
+      priority = priority.trim();
+    }
```

Comment:
@panagiotisgts This trims the string before writing it to the BPMN file, rather than trimming it during parsing. BPMN files may still have priority defined as:
- ` ` (single space)
- `abc` (some string)
- etc

Rather than dealing with it here, it may make more sense to add a runtime validator

https://github.com/camunda/camunda/blob/main/zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/deployment/model/validation/ZeebeRuntimeValidators.java